Code/Legacy/CryCommon/LegacyAllocator.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
```

Comment:
`#pragma once` has to be removed from the file(Clang will fail to build)

Reply:
good catch, copy paste fail